MonstieWash/Assets/Scripts/MonsterAi/MonsterBrain.cs

```diff
@@ -0,0 +1,308 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class MonsterBrain : MonoBehaviour
+{
+
+    [Tooltip("Add all moodtype objects intended for this brain here.")] [SerializeField] private List<MoodType> moodData; //Scriptable objects holding data about moods.
+
+    private Dictionary<int,float> activeMoods; //Current moods status. int refers to id and number of mood in list, float refers to current value of mood on its own scale.
+
+    private Dictionary<string, int> activeMoodNames; // Current moods and their names. int refers to id and number of mood in list, string refers to name.
+
+    [Tooltip("Updates the debug window when turned on")] [SerializeField] public bool Debug;
+
+
+    [Tooltip("Pauses the brain when on.")][SerializeField] public bool Pause;
+
+    private int designerSanityBuff; // A multiplier to reduce the tiny size of numbers used in setting up scriptable objects. Recommended set at 10.
```

Comment:
small thing, just because it's completely private now it should be named with an 'm_' prefix.
```suggestion
    private int m_designerSanityBuff; // A multiplier to reduce the tiny size of numbers used in setting up scriptable objects. Recommended set at 10.
```